sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -573,6 +591,18 @@ public void testDatabaseMetaDataTables() throws SQLException
                 Pair.of("TABLE_SCHEM", "druid"),
                 Pair.of("TABLE_TYPE", "TABLE")
             ),
+            row(
+                Pair.of("TABLE_CAT", "druid"),
+                Pair.of("TABLE_NAME", CalciteTests.T_ALL_TYPE_PARQUET),
+                Pair.of("TABLE_SCHEM", "druid"),
+                Pair.of("TABLE_TYPE", "TABLE")
+            ),
```

Comment:
I don't really see the benefit of loading all those drill related datasets for every test; they are not used in other tests

Reply:
Have made the change to load these datasources only for the drill test scenarios, have reverted these changes as well.